packages/cashier/src/components/side-notes/side-note-faq/side-note-faq.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { Accordion, SideNote, StaticUrl, Text } from '@deriv/components';
+import { LegacyChevronDown1pxIcon, LegacyChevronUp1pxIcon } from '@deriv/quill-icons';
+import { localize, Localize } from '@deriv/translations';
+import './side-note-faq.scss';
+
+type TSideNoteFAQProps = {
+    is_deposit?: boolean;
+    is_withdraw?: boolean;
+};
+
+const SideNoteFAQ = ({ is_deposit = false, is_withdraw = false }: TSideNoteFAQProps) => {
+    const onClickHandler = () => window.LC_API?.open_chat_window?.();
+
+    return (
+        <SideNote
+            description={
+                <Text size='xxs' weight='bold'>
+                    <Localize i18n_default_text='FAQ' />
+                </Text>
+            }
+        >
+            {is_deposit && (
+                <Accordion
+                    className='side-note-faq__accordion'
+                    icon_close={<LegacyChevronDown1pxIcon iconSize='xs' />}
+                    icon_open={<LegacyChevronUp1pxIcon iconSize='xs' />}
+                    list={[
+                        {
+                            header: localize("Why can't I see deposited funds in my Deriv account?"),
+                            content: (
+                                <Localize
+                                    i18n_default_text="We process your deposits internally within 24 hours (depending on internal checks and your payment method). If you don't see your funds reflected after this time, please contact us via <0>live chat</0> with proof of your transaction, including the amount, date, and time."
+                                    components={[
+                                        <span
+                                            key={0}
+                                            className='link link--orange'
```

Comment:
nit: I see that we have a classname called `link--orange` but we don't have any corresponding style definitions for it. Also, non of the links in the design are orange/red.
Is this required?

Reply:
I'll remove `link--orange` then. For the second point, as per the design team, we will keep the old style for links which is in red.